app/models/form/setup/questions/scheme_id.rb

```diff
@@ -22,7 +22,7 @@ def answer_options
   def displayed_answer_options(case_log)
     return {} unless case_log.created_by
 
-    user_org_scheme_ids = Scheme.select(:id).where(owning_organisation_id: case_log.created_by.organisation_id).map(&:id)
+    user_org_scheme_ids = Scheme.select(:id).where(owning_organisation_id: case_log.created_by.organisation_id).joins(:locations).merge(Location.where("startdate <= ? or startdate IS NULL", Time.zone.now)).map(&:id)
```

Comment:
Should we be comparing to a date here rather than a time?

Reply:
Time.zone.today is the closest I could get with the rubicon

Reply:
Why is it that my Mac thinks of rubocop as a soft drink or an existential concept?